modules/statics/src/coins.ts

```diff
@@ -555,6 +558,7 @@ export const coins = CoinMap.fromCoins([
   erc20('prints', 'FingerprintsDAO', 18, '0x4dd28568d05f09b02220b09c2cb307bfd837cb95', UnderlyingAsset.PRINTS),
   erc20('pro', 'Propy', 18, '0x9041fe5b3fdea0f5e4afdc17e75180738d877a01', UnderlyingAsset.PRO),
   erc20('prts', 'Protos', 0, '0x835a44027ee4e92bbd8874e5ede9e5148b069e96', UnderlyingAsset.PRTS),
+  erc20('pstake', 'PSTAKE', 18, '0xfb5c6815ca3ac72ce9f5006869ae67f18bf77006', UnderlyingAsset.PSTAKE),
```

Comment:
Would the full name for this one be pStake Finance?

Reply:
Im base in ticket as requested [https://bitgoinc.atlassian.net/browse/STLX-14864] but yes in etherscan is as you said https://etherscan.io/token/0xfB5c6815cA3AC72Ce9F5006869AE67f18bF77006#readContract. we should check with daniel wichone wants

Reply:
I'm going to change to pstake finance